### PR TITLE
✨ v1alpha3

### PR DESCRIPTION
**Pull request by akutz**
**What does this PR do, and why is it needed?**

This patch introduces schema version v1alpha3. This patch only includes plumbing changes for the API definitions, which, at this time, are a one-for-one copy of v1alpha2. This change allows developers to make updates to the API on main without impacting v1alpha2, which is locked.

**Which issue(s) is/are addressed by this PR?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes: NA

**Are there any special notes for your reviewer**:

**Please add a release note if necessary**:

```release-note
Introduce v1alpha3 as one-for-one copy of v1alpha2
```

----
📚 Documentation preview 📚: https://vm-operator--467.org.readthedocs.build/en/467/

### CODE REVIEW

File: api/v1alpha1/virtualmachineimage_conversion_test.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/vmware-tanzu/vm-operator/api/v1alpha1"
-	nextver "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
-	nextver_common "github.com/vmware-tanzu/vm-operator/api/v1alpha2/common"
+	nextver "github.com/vmware-tanzu/vm-operator/api/v1alpha3"
```
Comment (bryanv) [inline api/v1alpha1/virtualmachineimage_conversion_test.go:18]: Nit - there's still several comments mentioning `"v1a2"` in the conversion & conversion test files. Sub that with something like v1a2+, v1a3 or nextver or whatever

**Review by bryanv — APPROVED** [review]

File: pkg/vmprovider/providers/vsphere/vmprovider_vm.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 	"sync"
 	"text/template"
 
-	"github.com/vmware-tanzu/vm-operator/pkg/vmprovider/providers/vsphere2/session"
+	"github.com/vmware-tanzu/vm-operator/pkg/vmprovider/providers/vsphere/session"
```
[the remark this answers is not in the document]
Comment (akutz) [reply pkg/vmprovider/providers/vsphere/vmprovider_vm.go:14]: It is sorted in the follow-up flattening one.

File: api/v1alpha1/virtualmachineimage_conversion_test.go (hunk 1 above)
Comment (akutz) [reply api/v1alpha1/virtualmachineimage_conversion_test.go:18]: Will do.

File: pkg/vmprovider/providers/vsphere/vmprovider_vm.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (akutz) [reply pkg/vmprovider/providers/vsphere/vmprovider_vm.go]: Since I had to repush this anyway, I fixed it.

**Pull request merged**